manifests/n/NordSecurity/NordLocker/5.3.11.0/NordSecurity.NordLocker.installer.yaml

```
@@ -0,0 +1,32 @@
+# Created with komac v2.13.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: NordSecurity.NordLocker
+PackageVersion: 5.3.11.0
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Protocols:
+- nordlocker
+- nordlockerauth
+FileExtensions:
+- locker
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+ProductCode: '{BF7B051B-2E2E-4E1D-B1A3-D58723D30E1E}_is1'
+ReleaseDate: 2025-10-27
+AppsAndFeaturesEntries:
+- Publisher: nordvpn s.a.
+  ProductCode: '{BF7B051B-2E2E-4E1D-B1A3-D58723D30E1E}_is1'
+ElevationRequirement: elevatesSelf
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\NordLocker'
+Installers:
+- Architecture: x64
+  # InstallerUrl: https://downloads.nordcdn.com/apps/windows/NordLockerEvolution/NordLockerSetup_x64.exe
+  InstallerUrl: https://downloads.nordcdn.com/apps/windows/NordLockerEvolution/NordLockerSetup_x64.exe
```

```suggestion
  InstallerUrl: https://downloads.nordcdn.com/apps/windows/NordLockerEvolution/5.3.11.0/NordLockerSetup_x64.exe
```

Pin URL to version-specific one.